src/Symfony/Bundle/SecurityBundle/SecurityBundle.php

```diff
@@ -75,6 +76,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new RegisterCsrfFeaturesPass());
         $container->addCompilerPass(new RegisterTokenUsageTrackingPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, 200);
         $container->addCompilerPass(new RegisterLdapLocatorPass());
+        $container->addCompilerPass(new SortFirewallListenersPass(), PassConfig::TYPE_BEFORE_REMOVING);
```

Comment:
I think it's good to always add a comment when the priority is not the default. It is done with a reason, but that reason is often hidden and not visible when reading the code.
(I know that this is far from common in Symfony code, but I believe it's something Symfony source can improve a lot)

Reply:
Done :)